demisto_sdk/commands/download/downloader.py

```diff
@@ -511,7 +522,7 @@ def merge_and_extract_new_file(self, custom_content_object: dict) -> None:
         dir_output_path: str = os.path.join(self.output_pack_path, file_entity)
         # dir name should be the same as file name without separators mentioned in constants.py
         dir_name: str = self.create_dir_name(file_name)
-        dir_output_path: str = os.path.join(dir_output_path, dir_name)
+        dir_output_path = os.path.join(dir_output_path, dir_name)
```

Comment:
why did you change this? whats returned from the func?

Reply:
Double annotation.
![image](https://user-images.githubusercontent.com/11165655/80954886-ecae1100-8e06-11ea-8895-4a48d1098eca.png)